### PR TITLE
Added object transformation function.

### PR DESCRIPTION
**Pull request by cermak-petr**
Added an optional attribute in the input JSON containing a text repesentation of a function for transforming objects before importing them into the database.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -29,6 +29,13 @@ const importObjectToCollection = async (collection, object, importStats, uniqueK
     sleepPromised(100);
 };
 
+const includeTransformFunction(fnText){
```
Comment (drobnikj) [inline main.js]: Did you commit right version?
This probably doesn't work and you do not use includeTransformFunction anywhere.

File: main.js [hunk 2]
```diff
@@ -50,12 +57,22 @@ Apify.main(async () => {
 
     const uniqueKeys = input.uniqueKeys;
     const timestampAttr = input.timestampAttr;
+    
+    if (input.transformFunction) {
+        eval(input.transformFunction);
+        if (!transform || (typeof transform != "function")) {
+            throw new Error('Transform function is not correctly defined! Please consult readme.');
```
Comment (drobnikj) [inline main.js]: Duplicate code, can we checked if exists once or put this code in function.

**Review by drobnikj — COMMENTED** [review]
Cus @cermak-petr mrkni na komenty.

File: main.js (hunk 1 above)
Comment (cermak-petr) [reply main.js]: Yes, this is from old version, I don't know how it got here. I'll try removing it.

File: main.js (hunk 2 above)
Comment (cermak-petr) [reply main.js]: It should be correct now.

File: main.js [hunk 3]
```diff
@@ -50,12 +50,22 @@ Apify.main(async () => {
...
+        }
+    }
 
+    const processObject = transform || object => object;
```
Comment (drobnikj) [inline main.js]: cuz, tohle jeste padne na

/Users/Drobnik/WebstormProjects/apify-act-mongodb-import/main.js:61
    const processObject = transform || object => object;
                                    ^^

SyntaxError: Unexpected token ||
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:607:28)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3

můžeš to otestovat z lokálunormálně pomocí `npm run local`
stačí jen vytvořit kvs-local složku a v ní INPUT soubor

**Pull request merged**